subprojects/core/src/integTest/groovy/org/gradle/api/tasks/AbstractLineEndingNormalizationIntegrationSpec.groovy

```diff
@@ -204,26 +315,34 @@ abstract class AbstractLineEndingNormalizationIntegrationSpec extends AbstractIn
         return true
     }
 
-    def createTaskWithNormalization(LineEndingNormalization normalization, PathSensitivity pathSensitivity, API api) {
+    def createTaskWithNormalization(Class<?> inputAnnotation, LineEndingNormalization normalization, PathSensitivity pathSensitivity, API api) {
         buildFile << """
             task taskWithInputs(type: TaskWithInputs)
         """
 
         if (api == API.ANNOTATION_API) {
-            createAnnotatedTaskWithNormalization(normalization, pathSensitivity)
+            createAnnotatedTaskWithNormalization(inputAnnotation, normalization, pathSensitivity)
         } else if (api == API.RUNTIME_API) {
-            createRuntimeApiTaskWithNormalization(normalization, pathSensitivity)
+            Class<?> normalizer;
+            if (inputAnnotation == Classpath) {
+                normalizer = ClasspathNormalizer
+            } else if (inputAnnotation == CompileClasspath) {
+                normalizer = CompileClasspathNormalizer
+            } else {
+                normalizer = null
+            }
+            createRuntimeApiTaskWithNormalization(normalization, pathSensitivity, normalizer)
```

Comment:
Using both `normalization` and `normalizer` is confusing.

Reply:
It's now `normalizer` and `lineEndingSensitivity`.